File: JavaScript/Pattern4.js
class Num_pyramid{
    Pattern4(N){
        for(let i=1; i<=N; i++){
            let row = "";
            for(let j =1; j<=i; j++){
                row += i + " ";
            }
            console.log(row);
        }
           
    }
}

const sol = new Num_pyramid();
const N = 5;
sol.Pattern4(N)

File: Python/Pattern4.py
class Num_pyramid:
    def Pattern4(self, N):
        for i in range(1, N+1):
            for j in range(1, i + 1):
                print(i, end = " ")
            print()
    
if __name__ == "__main__":
    sol = Num_pyramid()
    N=5
    sol.Pattern4(N)
